pkg/ui/src/views/shared/components/summaryBar/index.tsx

```diff
@@ -6,6 +6,7 @@ import "./summarybar.styl";
 
 import { MetricsDataProvider } from "src/views/shared/containers/metricDataProvider";
 import { MetricsDataComponentProps } from "src/views/shared/components/metricQuery";
+import {ToolTipWrapper} from "oss/src/views/shared/components/toolTip";
```

Comment:
Please only prefix imports with `oss` if necessary to resolve an OSS component from CCL code.  The general rule should be to just write imports starting with `src`.

Also, spaces within braces please.

Reply:
Got it; will fix next time I touch this.